pkg/resourcecollector/resourcecollector.go

```diff
@@ -62,6 +69,9 @@ func (r *ResourceCollector) Init() error {
 	if err != nil {
 		return err
 	}
+
+	// reset k8s instance to given cluster config
+	k8s.Instance().SetConfig(config)
```

Comment:
We can't do this since that updates the singleton and it'll cause all the sched-ops APIs to go to the remote cluster if config is passed in.

We need to use `k8s.NewInstance()` instead and use that in GetResources, and initialize the clients in `GetResources`

Reply:
we use `k8s.Instance()` while collecting resources for eg. `pvcResourceCollector`